### PR TITLE
GitHub integration

### PR DESCRIPTION
**Pull request by msz-dev**
This regards implementation of the git hub integration.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -58,3 +58,6 @@ gem "sidekiq"
 
 gem 'redis-namespace'
 
+gem "github_api"
```
Comment (gvillalta99) [inline Gemfile]: @msz-dev what is this for?

File: app/controllers/authors_controller.rb [hunk 2]
```diff
@@ -55,6 +55,11 @@ def report
     redirect_to @author
  end
 
+  def login
+    @credentials = "msz-dev:localhost8080#@!"
```
Comment (gvillalta99) [inline app/controllers/authors_controller.rb]: Hey @msz-dev, are these real credentials?

Comment (gvillalta99) [inline app/controllers/authors_controller.rb]: Is this action being used?

File: app/controllers/home_controller.rb [hunk 3]
```diff
@@ -1,4 +1,19 @@
...
-  # This is the index action on the controller HomeController
+  
   def index; end
+
+  def auth
+    @code = params[:code]
+    @client_id = ENV['CLIENT_ID']
+    @client_secret = ENV['CLIENT_SECRET']
+    @response = RestClient.post("https://github.com/login/oauth/access_token", {client_id:@client_id, client_secret:@client_secret, code:@code })
+    @body = Hash[@response.body.split('&').map{|element| element.split '='}]
```
Comment (gvillalta99) [inline app/controllers/home_controller.rb]: What is going on here?

Comment (msz-dev) [inline app/controllers/home_controller.rb]: This is the "happy path" to make the auth on the git hub. I was wondering if there is a best way to do it or some best pratices to follow.

**Pull request merged**